EKF/drag_fusion.cpp

```diff
@@ -88,114 +86,173 @@ void Ekf::fuseDrag()
 		const float mea_acc = _drag_sample_delayed.accelXY(axis_index)  - _state.delta_vel_bias(axis_index) / _dt_ekf_avg;
 		const float airSpd = sqrtf((2.0f * fabsf(mea_acc)) / (ballistic_coef_inv_xy(axis_index) * rho));
 
+		// Estimate the derivative of specific force wrt airspeed along the X axis
```

Comment:
Do you want to make the above introduced variables vn,ve,vd,vwn,vwe a reference as with the other fusion types?